iOS/DuckDuckGo/AppLifecycle/AppDependencyProvider.swift

```diff
@@ -117,7 +117,14 @@ final class AppDependencyProvider: DependencyProvider {
         var authenticationStateProvider: (any SubscriptionAuthenticationStateProvider)!
 
         let tokenStorageV2 = SubscriptionTokenKeychainStorageV2(keychainType: .dataProtection(.named(subscriptionAppGroup))) { keychainType, error in
-            Pixel.fire(.privacyProKeychainAccessError, withAdditionalParameters: ["type": keychainType.rawValue, "error": error.errorDescription])
+
+            let parameters = [PixelParameters.privacyProKeychainAccessType: keychainType.rawValue,
```

Comment:
`keychainType` has a wrong name, it should rather be `accessType`

Reply:
the keychainType parameter matches the KeychainType enum, not my decision. Not super keen to change it across 21 uses.

Reply:
I am not saying about the enum type, but the variable name in the closure. The enum is called `AccountKeychainAccessType` and represents not the keychain type but type of access to the keychain (e.g. was it get or store etc.). 

I am asking to replace all of `{ keychainType, error in` -> `{ accessType, error in`

Reply:
changed